models/DataObject/Localizedfield/Dao.php

```diff
@@ -167,7 +167,7 @@ public function save($params = [])
                             )
                             || (((!$container instanceof DataObject\Fieldcollection\Definition || $container instanceof DataObject\Objectbrick\Definition)
                                     && $this->model->isLanguageDirty($language))
-                                || $params['saveRelationalData']['saveLocalizedRelations'])) {
+                                || ($params['saveRelationalData']['saveLocalizedRelations'] ?? false))) {
```

Comment:
We could simplify the if when we wrote that in a variable beforehand:
`$saveLocalizedRelations = $params['saveRelationalData']['saveLocalizedRelations'] ?? false;`

What do you think?

Reply:
@weisswurstkanone It's not the only line. You could also save the isset calls 6/7 lines above...